libraries/plugins/tags/include/scorum/tags/tags_api_impl.hpp

```diff
@@ -433,6 +433,49 @@ class tags_api_impl
         return result;
     }
 
+    posts_crefs exclude(const posts_crefs& filtering_posts,
+                        const discussion_query& query,
+                        const std::function<bool(const tag_object&)>& tag_filter) const
+    {
+        auto rng_exclude = query.exclude_tags | boost::adaptors::transformed(utils::to_lower_copy)
```

Comment:
Add `//` (comment) just after `auto rng_exclude = query.exclude_tags` to force clang-format to move each `| boost::adaptors` on separate line